### PR TITLE
adding test delays for flakiness & fixing null error in buildWalletSection

### PR DESCRIPTION
**Pull request by skylarbarrera**

### CODE REVIEW

File: src/helpers/buildWalletSections.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ const addEth = section => {
           amount: '0',
           display: '$0',
         },
-        change: `${relative_change_24h.toFixed(2)}%`,
+        change: `${relative_change_24h?.toFixed(2)}%`,
```
Comment (brunobar79) [inline src/helpers/buildWalletSections.js:103]: This fixes the crash but we might show "%" if it's falsey.

Unless I'm wrong and we're checking that somewhere else, would be better if we check and return an empty string instead.

**Review by brunobar79 — APPROVED** [review]
All good in the hood
